src/main/java/build/buildfarm/common/ExecutorServices.java

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build.buildfarm.common;
+
+import static com.google.common.util.concurrent.MoreExecutors.listeningDecorator;
+
+import com.google.common.util.concurrent.ListeningExecutorService;
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * @class ExecutorServices
+ * @brief Group executor services to easier manage thread counts across software stack.
+ * @details Thread usage is a global concern across the java application. Although these thread
+ *     pools are used in different contexts we group them here so the application's threads can be
+ *     managed and derived from the same class.
+ */
+public class ExecutorServices {
+  public static ExecutorService getScanCachePool() {
```

Comment:
Since all of these create new Executors, please change the name pattern to either `create` or `new` (the latter probably being more fitting).